src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,9 +6,9 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_NAME = new Prefix("name/");
+    public static final Prefix PREFIX_PHONE = new Prefix("contact/");
+    public static final Prefix PREFIX_GENDER = new Prefix("gender/");
```

Comment:
Actually, I think it might be good to use the shorter versions? More user-friendly! We can update the UG to reflect this if everyone agrees